cordova-lib/spec-plugman/plugins/org.test.defaultvariables/plugin.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<plugin xmlns="http://cordova.apache.org/ns/plugins/1.0"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    id="org.test.defaultvariables"
+    version="3.0.0">
+
+    <name>Use Default Variables</name>
+
+    <!-- android -->
+    <platform name="android">
+            <preference name="ENABLE_LOG" default="yes" />
+            <preference name="API_KEY" />
+    </platform>
```

Comment:
I only found the default `ENABLE_LOG` in `cordova/plugins/android.json`, but there is no similar preference `<preference name="ENABLE_LOG" value="yes" />` in config.xml, how to use the Default variable.